Util/CellUtils/cell2str.m

```diff
@@ -0,0 +1,67 @@
+function [cstr] = cell2str(xcell, sep),
+    % function [cstr] = cell2str(xcell,sep),
+    %
+    % Convert all items in a cell to string and concatenate
+    % then with a separator, ignoring non-numeric entries.
+    %
+    % Inputs:
+    %
+    % xcell - a cell
+    % sep - a string separator - default to ','
+    %
+    % Outputs:
+    %
+    % cstr - a string of all strings and numerics separated by
+    %        `sep`.
+    %
+    % Example:
+    % cstr = cell2str({1,'string'})
+    % assert(strcmp(cstr,'1,string'))
+    %
+    % author: hugo.oliveira@utas.edu.au
+    %
+
+    % Copyright (C) 2019, Australian Ocean Data Network (AODN) and Integrated
+    % Marine Observing System (IMOS).
+    %
+    % This program is free software: you can redistribute it and/or modify
+    % it under the terms of the GNU General Public License as published by
+    % the Free Software Foundation version 3 of the License.
+    %
+    % This program is distributed in the hope that it will be useful,
+    % but WITHOUT ANY WARRANTY; without even the implied warranty of
+    % MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    % GNU General Public License for more details.
+    %
+    % You should have received a copy of the GNU General Public License
+    % along with this program.
+    % If not, see <https://www.gnu.org/licenses/gpl-3.0.en.html>.
+    %
+
+    if nargin < 2,
+        sep = ',';
+    end
+
+    if ~isstr(sep),
+        error('second argument must be string')
+    end
+
+    cn = numel(xcell);
+    cstr = '';
+    ksep = '';
+
+    for k = 1:cn,
+        data = xcell{k};
+
+        if isnumeric(data),
+            cstr = [cstr ksep num2str(data)];
+        elseif isstr(data),
+            cstr = [cstr ksep data];
+        else
+            disp('Ignored entry %d in cell', k)
```

Comment:
I've tried to uniform the format of any message displayed so that a user knows whether this is info, warning or error. See  `./AutomaticQC/imosSideLobeVelocitySetQC.m` for example.
```Matlab
disp(['Info : imosSideLobeVelocitySetQC uses nominal depth because no pressure data in file ' sample_data.toolbox_input_file]);
```

Reply:
Thanks - the line is actually wrong, disp only accepts one argument.

Reply:
I provided a new function - swarning - in the commit below. It takes care of knowing the scope so we don't need to provide the name of the function as a string in the argument. The function reports the full scope in the call chain (a la error).